### PR TITLE
Indirect Example Workflows for Data Analysis and Reduction

### PR DESCRIPTION
**Pull request by robertapplin**
**Description of work.**
This PR introduces updated and improved documented workflows for Elwin, MSDFit, Iqt, IqtFit, ConvFit and F(Q)Fit.
This PR also introduces simple workflows for ISIS Energy Transfer, ISIS Calibration, ISIS Diagnostics, Transmission, Symmetrise, S(Q,w) and Moments.

**To test:**
Read the documentation and attempt the workflows. Check the links work.

Part Of #24685 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by TTitcombe — DISMISSED** [review]
Looks good :+1: Instructions are clear and concise. Links in documentation work

File: docs/source/interfaces/Indirect Data Analysis.rst [hunk 1]
```diff
@@ -449,6 +597,39 @@ Edit Result
 Save Result
   Saves the workspaces from the *_Results* group workspace in the default save directory.
 
+.. _convfit-example-workflow:
+
+ConvFit Example Workflow
+~~~~~~~~~~~~~~~~~~~~~~~~
+The Conv Fit tab allows ``_red`` and ``_sqw`` for it's sample file, and allows ``_red``, ``_sqw`` and
```
Comment (TTitcombe) [inline docs/source/interfaces/Indirect Data Analysis.rst]:
```suggestion
The Conv Fit tab allows ``_red`` and ``_sqw`` for its sample file, and allows ``_red``, ``_sqw`` and
```

**Review by TTitcombe — CHANGES_REQUESTED** [review]

**Comment by TTitcombe** [comment]
@robertapplin I don't know what you've done to the Git history, but your PR now includes 103 files

**Comment by robertapplin** [comment]
@TTitcombe I have no idea what I did either but it should now be fixed 👍

**Review by TTitcombe — APPROVED** [review]

**Comment by TTitcombe** [comment]
lgtm :+1: Clear documentation on Indirect workflow

**Pull request merged**